sqlx/sqlx_service.c

```diff
@@ -543,16 +555,25 @@ sqlx_service_action(void)
 			return _action_report_error(err, "Failed to start the QUEUE thread");
 	}
 
-	/* SERVER/GRIDD main run loop */
+	/* open all the sockets */
 	if (!grid_main_is_running())
 		return;
+	if (udp_allowed)
```

Comment:
I think it's harmless to enable UDP server by default. This would simplify the migration path: when you know all your servers have been updated, you can enable UDP clients.